script/release.sh

```diff
@@ -139,13 +139,20 @@ CONTRIBUTORS=$(git log "${LAST_RELEASE_VERSION}".. --format="%aN" --reverse | so
 CHANGELOG=$(git log "${LAST_RELEASE_VERSION}".. --oneline | grep -v 'Merge pull request')
 
 CHECKSUM=""
-for file in $(ls bin/docker-machine*); do
+for file in $(ls bin/docker-machine-*); do
   SHA256=$(openssl dgst -sha256 < "${file}")
   MD5=$(openssl dgst -md5 < "${file}")
   LINE=$(printf "\n * **%s**\n  * sha256 \`%s\`\n  * md5 \`%s\`\n\n" "$(basename ${file})" "${SHA256}" "${MD5}")
   CHECKSUM="${CHECKSUM}${LINE}"
 done
 
+# Generate Checksum file
+(
+    cd bin
+    shasum -a 256 docker-machine-* > ../sha256sum.txt
```

Comment:
If replace the `shasum -a 256` with `openssl dgst -sha256` the checksum format will be different.

```bash
shasum -a 256 docker-machine-* > ../sha256sum.txt
```

will generate result

```bash
3fb225ece68c6517482a6109e737f77dda000ee4d35685e1977f7ed0f4587ef7  docker-machine-Darwin-x86_64
c844d3d37a677ab660fd9f14d7fd0fe1b323ae12ee33d8cbe73907a274d530ab  docker-machine-Linux-armhf
c6e35da73d81c5fc0236e786696397b414a41df8580cf36f8f1f27527222daa9  docker-machine-Linux-x86_64
2d1725b06bafc77ba14a2794696d03f2d29c85984d3211069f2d2ee2c6278a70  docker-machine-Windows-i386.exe
91ec3ad748740d125efc15ce08a454d0d9c03ecf465ee3ec3ff50a98716cc33c  docker-machine-Windows-x86_64.exe
```

And with

```bash
openssl dgst -sha256 docker-machine-* > ../sha256sum.txt
```

will generate the checksum in following format:

```bash
SHA256(docker-machine-Darwin-x86_64)= 3fb225ece68c6517482a6109e737f77dda000ee4d35685e1977f7ed0f4587ef7
SHA256(docker-machine-Linux-armhf)= c844d3d37a677ab660fd9f14d7fd0fe1b323ae12ee33d8cbe73907a274d530ab
SHA256(docker-machine-Linux-x86_64)= c6e35da73d81c5fc0236e786696397b414a41df8580cf36f8f1f27527222daa9
SHA256(docker-machine-Windows-i386.exe)= 2d1725b06bafc77ba14a2794696d03f2d29c85984d3211069f2d2ee2c6278a70
SHA256(docker-machine-Windows-x86_64.exe)= 91ec3ad748740d125efc15ce08a454d0d9c03ecf465ee3ec3ff50a98716cc33c
```

And `openssl dgst` cannot verify the checksum file it generated, unless it's been signed. So, I think it's better keeping `shasum` and `md5sum` format for checksum files, so user can verify the downloaded files simply by `shasum -a 256 -c sha256sum.txt`.

And to keep the same format of `md5sum` and `shasum` generated checksum file, I think it's better to use the `shasum` and `md5sum` for the job. I installed `md5sum` on my macOS via `brew install md5sha1sum`.

Do you think it is necessary to remove the `md5sum` dependency? If so, I think I might be able to construct the checksum file from the checksum calculated in the above section, such as:

```bash
CHECKSUM=""
rm sha256sum.txt md5sum.txt
for file in $(ls bin/docker-machine-*); do
  SHA256=$(openssl dgst -sha256 < "${file}")
  MD5=$(openssl dgst -md5 < "${file}")
  LINE=$(printf "\n * **%s**\n  * sha256 \`%s\`\n  * md5 \`%s\`\n\n" "$(basename ${file})" "${SHA256}" "${MD5}")
  CHECKSUM="${CHECKSUM}${LINE}"
  echo "${SHA256}  ${file:4}" >> sha256sum.txt
  echo "${MD5}  ${file:4}" >> md5sum.txt
done
```

What do you think?